test/Microsoft.Azure.Devices.Edge.Test/IoTEdgeCheck.cs

```diff
@@ -48,6 +48,9 @@ public async Task IotEdgeCheck()
                 });
             }
 
+            // TODO: Temporarily override expected aziot & iotedge versions until after we release 1.5.0
+            args += " --expected-aziot-version 1.5.0 --expected-aziot-edged-version 1.5.0";
```

Comment:
Let's create a PBI to follow up on this so we don't forget about it.

Reply:
Agreed. I also hard-coded to 1.5.0 so that this will start failing if we forget to change it and 1.5.1 ships.